### PR TITLE
hosts.update: Reconfigure CentOS 8 repositories to vault

### PR DESCRIPTION
**Pull request by anoopcs9**
CentOS Stream 8 reached EOL but our GPFS and GlusterFS environment setup still has some dependencies which are not yet fulfilled by CentOS Stream 9. In case of GlusterFS it lacks few gluster-ansible related rpms whereas GPFS is not yet ready to be compiled against newer kernel versions from CentOS Stream 9.

Therefore we stick to CentOS Stream 8 till requirements are met. See https://github.com/samba-in-kubernetes/sit-environment/issues/102 for further developments.

### CODE REVIEW

File: playbooks/roles/hosts.update/tasks/centos8.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+# https://github.com/samba-in-kubernetes/sit-environment/issues/102
+- name: Modify repositories to point CentOS vault
```
Comment (phlogistonjohn) [inline playbooks/roles/hosts.update/tasks/centos8.yml:3]: minor grammar issue: either 'point at CentOS vault' or 'point to CentOS vault'

Comment (anoopcs9) [reply playbooks/roles/hosts.update/tasks/centos8.yml]: Done.

**Review by xhernandez — DISMISSED** [review]

**Review by phlogistonjohn — DISMISSED** [review]
lgtm, thanks.

**Review by Shwetha-Acharya — APPROVED** [review]

**Comment by anoopcs9** [comment]
GlusterFS job failed because of an extra CentOS Storage SIG repository enabled(via installation of _centos-release-gluster_) later during execution of backend specific tasks for installing some dependencies(may be only for _glusterfs-selinux_). We'll have to add similar change exclusively inside _playbooks/ansible/roles/sit.glusterfs/tasks/common_. I'll update the PR.

**Review by xhernandez — APPROVED** [review]

**Pull request merged**